wdio/scripts/runAndroidTests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Start emulator and run tests
+emulator @Pixel_8.1 & ./node_modules/.bin/wdio test/configs/android.conf.js
```

Comment:
wonder if you can grab the first installed emulator by doing something like this `emulator -list-avds | head -n 1`

Reply:
We have a ticket where we are going to validate installed sims and install missing ones as necessary.